spring-cloud-gcp-storage/src/main/java/org/springframework/cloud/gcp/storage/GoogleStorageProtocolResolver.java

```diff
@@ -64,6 +64,6 @@ public Resource resolve(String location, ResourceLoader resourceLoader) {
 		if (location.startsWith("gs://")) {
 			return new GoogleStorageResource(this.storage, location);
 		}
-		return this.delegate.getResource(location);
+		return null;
```

Comment:
OK. So the code in the `DefaultResourceLoader` looks like:
```
for (ProtocolResolver protocolResolver : this.protocolResolvers) {
	Resource resource = protocolResolver.resolve(location, this);
	if (resource != null) {
		return resource;
	}
}
```
So, that was a bug from day first do not return `null` in this `GoogleStorageProtocolResolver` impl. 😄